index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+    integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
+  <link rel="stylesheet" href="style.css">
+  <title>Yuriy's Portfolio</title>
+</head>
+
+<body>
+  <div class="container">
+    <header>
```

Comment:
[ OPTION BUT RECOMMENDED ] The Toolbar section should have a hamburger menu button on right. Please make sure the hamburger is shown here instead of the navigation list on Mobile view large
![menu](https://user-images.githubusercontent.com/35504149/138300536-903271e5-82c9-40ef-919d-cff8fefd6cbb.png)
.